addons/binding/org.openhab.binding.freemobilesms/README.md

```diff
@@ -0,0 +1,53 @@
+# FreeMobile SMS Binding
+
+The FreeMobile SMS binding povides support for sending SMS via [FreeMobile API](https://www.freenews.fr/freenews-edition-nationale-299/free-mobile-170/nouvelle-option-notifications-par-sms-chez-free-mobile-14817).
+
+## Supported Things
+
+This binding supports the following thing types:
+
+| Thing         | Thing Type | Description                                             |
+|---------------|------------|---------------------------------------------------------|
+| account       | Thing      | One callable account.                                   |
+
+## Discovery
+
+Accounts should be created manually.
+
+## Binding Configuration
+
+The binding does not have settings.
+Future releases can ad the possibility to change the base URL, to let the user follow possible changes in the API.
```

Comment:
Please remove this section. Developer TODOs can be added in a second file

Reply:
Moved to a new TODO file